2016-09/september-21.md

```diff
@@ -0,0 +1,49 @@
+## September 21 ([discuss](https://github.com/webpack/meeting-notes/pull/14))
+
+### Attendees
+
+* [Tobias Koppers](http://github.com/sokra)
+* [Sean Larkin](http://github.com/thelarkinn)
+* [Juho Vepsäläinen](http://github.com/bebraw)
+
+
+### Agenda
+
+#### Documentation Updates
+
+* Juho is going to start working on [antwar/antwar#103](https://github.com/antwarjs/antwar/issues/103) next week.
```

Comment:
Happy to send a spike port PR, its the least we could do for all that webpack has given us. Will have something ready probably in a couple weeks! 😁